File: csharp/MapleStory.OpenAPI/Src/Dto/Guild/GuildSkillDTO.cs
using Newtonsoft.Json;

namespace MapleStory.OpenAPI.Dto
{
    /// <summary>
    /// 길드 스킬 정보
    /// </summary>
    public class GuildSkillDTO
    {
        /// <summary>
        /// 스킬 명
        /// </summary>
        [JsonProperty("skill_name")]
        public string SkillName { get; set; }

        /// <summary>
        /// 스킬 설명
        /// </summary>
        [JsonProperty("skill_description")]
        public string SkillDescription { get; set; }

        /// <summary>
        /// 스킬 레벨
        /// </summary>
        [JsonProperty("skill_level")]
        public long SkillLevel { get; set; }

        /// <summary>
        /// 스킬 레벨 별 효과
        /// </summary>
        [JsonProperty("skill_effect")]
        public string SkillEffect { get; set; }

        /// <summary>
        /// 스킬 아이콘
        /// </summary>
        [JsonProperty("skill_icon")]
        public string SkillIcon { get; set; }
    }
}

File: java/src/main/java/dev/spiralmoon/maplestory/api/tms/dto/guild/GuildSkillDTO.java
package dev.spiralmoon.maplestory.api.tms.dto.guild;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * 公會技能
 */
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class GuildSkillDTO implements dev.spiralmoon.maplestory.api.common.dto.guild.GuildSkillDTO {

    /**
     * 技能名稱
     */
    @JsonProperty("skill_name")
    private String skillName;

    /**
     * 技能描述
     */
    @JsonProperty("skill_description")
    private String skillDescription;

    /**
     * 技能等級
     */
    @JsonProperty("skill_level")
    private long skillLevel;

    /**
     * 技能等級的效果
     */
    @JsonProperty("skill_effect")
    private String skillEffect;

    /**
     * 技能圖示
     */
    @JsonProperty("skill_icon")
    private String skillIcon;
}
